### PR TITLE
A vanilla intro to pedersen commitments

### PR DESCRIPTION
**Pull request by wisbech**
Pedersen commitment introduction and its feasability for obfuscating sensitive information of certificates on the registry

### CODE REVIEW

**Review by lauranolling — APPROVED** [review]

File: doc/pedersen_commitments/pedersen-commitments.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Pedersen Commitments
+
+In order to obfuscate the amount of energy that is being transferred, the amount of energy is encrypted using Pedersen commitments. The Pedersen commitment is a commitment scheme that allows the holder of the commitment to commit to a value without revealing the value itself. The commitment can be opened to reveal the value. The holder cannot open it to reveal another value. The commitment scheme is homomorphic in the sense that the product of two commitments will be a commitment to the sum of the two values.
```
Comment (MartinSchmidt) [inline doc/pedersen_commitments/pedersen-commitments.md:3]: And other values, like GSRN ;)

File: doc/pedersen_commitments/pedersen-commitments.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+graph TD
+    A[Commitment] --> B[Message]
+    A --> C[Randomness]
+```
+
+### Additional properties of Pedersen commitments
+There are commutation and homomorphism properties that are useful for the Pedersen commitment scheme. These properties are defined as follows:
+
+
+## What these properties can do in practice for Energy Track and Trace
```
Comment (MartinSchmidt) [inline doc/pedersen_commitments/pedersen-commitments.md:23]: ProjectOrigin?

File: doc/pedersen_commitments/pedersen-commitments.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    E --> |*| F
+    J --> |=| M
+    S5 --> C
+```
+
+### Pedersen commitments are used in the following way to obfuscate the amount of energy that is being transferred:
+We want to show that the sum of the commitments in a certificate (e.g. transactions + remainder) is equal to the committed value in the original production certificate, 
+corresponding to 
+
+$$Ctotal / (Ctransactions * Cremainder)$$
```
Comment (MartinSchmidt) [inline doc/pedersen_commitments/pedersen-commitments.md:72]: C´ or Commitment to 0 == ...

Maybe it should be more explicit that it is not normal devide and times, but simple the notation one uses.

**Review by MartinSchmidt — COMMENTED** [review]
i have added a few comments..

File: doc/pedersen_commitments/pedersen-commitments.md (hunk 2 above)
Comment (wisbech) [reply doc/pedersen_commitments/pedersen-commitments.md:23]: Yes we are indeed working on project origin.

**Pull request merged**